<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModelBuilder</class>
 <widget class="QMainWindow" name="ModelBuilder">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Personal library of models</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="WorkspaceView" name="listView">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="indentation">
           <number>4</number>
          </property>
          <property name="rootIsDecorated">
           <bool>false</bool>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="headerHidden">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="modelFile">
            <property name="text">
             <string>TextLabel</string>
            </property>
            <property name="textInteractionFlags">
             <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="buttonDescription">
            <property name="text">
             <string>Model Description</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Editor" name="editor" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
    <addaction name="actionEdit"/>
    <addaction name="actionSave"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../Resource.qrc">
     <normaloff>:/img/Resources/exit.png</normaloff>:/img/Resources/exit.png</iconset>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../Resource.qrc">
     <normaloff>:/img/Resources/save.png</normaloff>:/img/Resources/save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="../Resource.qrc">
     <normaloff>:/img/Resources/delete.png</normaloff>:/img/Resources/delete.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="../Resource.qrc">
     <normaloff>:/img/Resources/model-new.png</normaloff>:/img/Resources/model-new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="icon">
    <iconset resource="../Resource.qrc">
     <normaloff>:/img/Resources/open.png</normaloff>:/img/Resources/open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionCloseModel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Resource.qrc">
     <normaloff>:/Resources/exit.png</normaloff>:/Resources/exit.png</iconset>
   </property>
   <property name="text">
    <string>actionCloseModel</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="../Resource.qrc">
     <normaloff>:/Resources/help.png</normaloff>:/Resources/help.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WorkspaceView</class>
   <extends>QTreeView</extends>
   <header location="global">workspaceview.h</header>
  </customwidget>
  <customwidget>
   <class>Editor</class>
   <extends>QWidget</extends>
   <header>editor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>ModelBuilder</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>animate()</slot>
 </slots>
</ui>
